site-builder/src/site/content.rs

```diff
@@ -121,89 +121,92 @@ pub enum ContentType {
+            "jpeg" => Ok(ContentType::ImageJpeg),
+            "jpg" => Ok(ContentType::ImageJpeg),
+            "js" => Ok(ContentType::TextJavascript),
+            "json" => Ok(ContentType::ApplicationJson),
+            "jsonld" => Ok(ContentType::ApplicationLdjson),
+            "mid" => Ok(ContentType::AudioMidi),
+            "midi" => Ok(ContentType::AudioMidi),
+            "mjs" => Ok(ContentType::TextJavascript),
+            "mp3" => Ok(ContentType::AudioMpeg),
+            "mp4" => Ok(ContentType::VideoMp4),
+            "mpeg" => Ok(ContentType::VideoMpeg),
+            "mpkg" => Ok(ContentType::ApplicationVndappleinstallerxml),
+            "odp" => Ok(ContentType::ApplicationVndoasisopendocumentpresentation),
+            "ods" => Ok(ContentType::ApplicationVndoasisopendocumentspreadsheet),
+            "odt" => Ok(ContentType::ApplicationVndoasisopendocumenttext),
+            "oga" => Ok(ContentType::AudioOgg),
+            "ogv" => Ok(ContentType::VideoOgg),
+            "ogg" => Ok(ContentType::VideoOgg),
+            "ogx" => Ok(ContentType::ApplicationOgg),
+            "opus" => Ok(ContentType::AudioOpus),
+            "otf" => Ok(ContentType::FontOtf),
+            "png" => Ok(ContentType::ImagePng),
+            "pdf" => Ok(ContentType::ApplicationPdf),
+            "php" => Ok(ContentType::ApplicationXhttpdphp),
+            "ppt" => Ok(ContentType::ApplicationVndmspowerpoint),
+            "pptx" => Ok(
+                ContentType::ApplicationVndopenxmlformatsofficedocumentpresentationmlpresentation,
+            ),
+            "rar" => Ok(ContentType::ApplicationVndrar),
+            "rtf" => Ok(ContentType::ApplicationRtf),
+            "sh" => Ok(ContentType::ApplicationXsh),
+            "svg" => Ok(ContentType::ImageSvgxml),
+            "tar" => Ok(ContentType::ApplicationXtar),
+            "tif" => Ok(ContentType::ImageTiff),
+            "tiff" => Ok(ContentType::ImageTiff),
+            "ts" => Ok(ContentType::VideoMp2t),
+            "ttf" => Ok(ContentType::FontTtf),
+            "txt" => Ok(ContentType::TextPlain),
+            "vsd" => Ok(ContentType::ApplicationVndvisio),
+            "wav" => Ok(ContentType::AudioWav),
+            "weba" => Ok(ContentType::AudioWebm),
+            "webm" => Ok(ContentType::VideoWebm),
+            "webp" => Ok(ContentType::ImageWebp),
+            "woff" => Ok(ContentType::FontWoff),
+            "woff2" => Ok(ContentType::FontWoff2),
+            "xhtml" => Ok(ContentType::ApplicationXhtmlxml),
+            "xls" => Ok(ContentType::ApplicationVndmsexcel),
+            "xlsx" => Ok(ContentType::ApplicationVndopenxmlformatsofficedocumentspreadsheetmlsheet),
+            "xml" => Ok(ContentType::ApplicationXml),
+            "xul" => Ok(ContentType::ApplicationVndmozillaxulxml),
+            "zip" => Ok(ContentType::ApplicationZip),
+            "7z" => Ok(ContentType::ApplicationX7zcompressed),
+            _ => Err(anyhow!(
```

Comment:
Hint: If you change this to `return Err(...)`, you can remove the `Ok()` from all the other cases and just wrap the whole thing into an `Ok`.